### PR TITLE
[PLAY-741] Add "Copy Code" button to all kit code examples

### PR DESCRIPTION
**Pull request by augustomallmann**
Screenshot 2023-04-21 at 1 37 25 PM

**How to test?** Steps to confirm the desired behavior:
1. Go to any kit page and click on the "show code" button.

#### Checklist:
- [x] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `deprecated`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [x] **DEPLOY** I have added the `milano` label to show I'm ready for a review.
- [x] **TESTS** I have added test coverage to my code.

### CODE REVIEW

File: playbook-website/Gemfile.lock [hunk 1]
```diff
@@ -285,6 +285,7 @@ GEM
 PLATFORMS
   arm64-darwin-21
   arm64-darwin-22
+  x86_64-darwin-20
```
Comment (jasperfurniss) [inline playbook-website/Gemfile.lock:288]: Not sure why this got added.

Comment (jasperfurniss) [reply playbook-website/Gemfile.lock:288]: Yeah, but it shouldn't be adding this on this specific PR. A little odd. 
I'll run a commit and push up a quick commit to remove this.

File: playbook/app/pb_kits/playbook/playbook-rails.js [hunk 2]
```diff
@@ -9,6 +9,10 @@ window.datePickerHelper = datePickerHelper
 import dialogHelper from './pb_dialog/dialogHelper'
 window.dialogHelper = dialogHelper
 
+// CopyContent
+import copyContent from './pb_docs/copyCode'
```
Comment (jasoncypret) [inline playbook/app/pb_kits/playbook/playbook-rails.js:13]: This is used outside of docs. Please move

File: playbook/app/pb_kits/playbook/pb_docs/kit_example.html.erb [hunk 3]
```diff
@@ -13,16 +15,110 @@
...
+                class="svg-inline--fa fa-code pb_icon_kit fa-fw mr_xs" 
+                aria-hidden="true" 
+                focusable="false" 
+                data-prefix="far" 
+                data-icon="code" 
+                role="img" 
+                xmlns="http://www.w3.org/2000/svg" 
+                viewBox="0 0 640 512" 
+                data-fa-i2svg=""
+              >
```
Comment (jasoncypret) [inline playbook/app/pb_kits/playbook/pb_docs/kit_example.html.erb:76]: This is not great. What are we doing here.

File: playbook/app/pb_kits/playbook/playbook-rails.js (hunk 2 above)
Comment (augustomallmann) [reply playbook/app/pb_kits/playbook/playbook-rails.js]: this came from the [pr 2470](https://github.com/powerhome/playbook/pull/2470/files) that I merged to start working. I marged master to my branch and deleted this code.

File: playbook/app/pb_kits/playbook/pb_docs/kit_example.html.erb [hunk 4]
```diff
@@ -33,13 +33,36 @@
             </li>
           <% end %>
           <li>
-            <a href="#" id="toggle-open" data-toggle="code_example">Code Example</a>
+            <%= pb_rails("button", props: { 
+              text: "Show code", 
+              icon: "copy",
```
Comment (nidaqg) [inline playbook/app/pb_kits/playbook/pb_docs/kit_example.html.erb:38]: @jasperfurniss are we changing out this icon before we ship this? Do you know what the correct icon is?

Comment (jasperfurniss) [inline playbook/app/pb_kits/playbook/pb_docs/kit_example.html.erb:38]:
```suggestion
              icon: "code", 
```

**Review by nidaqg — DISMISSED** [review]

**Review by nidaqg — APPROVED** [review]

**Pull request closed without merging**